mappings/net/minecraft/entity/projectile/FishingBobberEntity.mapping

```diff
@@ -1,10 +1,17 @@
 CLASS net/minecraft/class_1536 net/minecraft/entity/projectile/FishingBobberEntity
 	CLASS class_1537 State
 	FIELD field_7165 hookedEntity Lnet/minecraft/class_1297;
-	FIELD field_7168 luckOfTheSeaLevel I
+	FIELD field_7166 selfHitTimer I
+	FIELD field_7167 removalTimer I
+	FIELD field_7168 lureLevel I
+	FIELD field_7169 fishAngleDegrees F
 	FIELD field_7170 HOOK_ENTITY_ID Lnet/minecraft/class_2940;
-	FIELD field_7171 lureLevel I
+	FIELD field_7171 luckOfTheSeaLevel I
+	FIELD field_7172 fishTravelCountdown I
+	FIELD field_7173 hookCountdown I
+	FIELD field_7174 waitCountdown I
 	FIELD field_7175 state Lnet/minecraft/class_1536$class_1537;
+	FIELD field_7176 hasHitNonEntity Z
```

Comment:
This name seems a bit unwieldy, and more appropriate for a getter than a boolean field.

Reply:
missedEntity?

Reply:
Ah, yeah. On closer look, `HitResult` can only be `MISS`, `ENTITY`, or `BLOCK`, so `hasHitBlock` would work. As for the wording, is `hitBlock`, `onBlock`, or `stuckOnBlock` an improvement?

Reply:
I think `stuckOnBlock` is the clearest one.